Example/Example/EditorDemoController.swift

```diff
@@ -1370,8 +1370,8 @@ private extension EditorDemoController
            detailsViewController.linkURL = url
         }
         
-        detailsViewController.onUpdate = { (alignment, size, url, linkURL, alt) in
-            self.richTextView.edit(attachment) { updated in
+        detailsViewController.onUpdate = { [weak self] (alignment, size, url, linkURL, alt) in
```

Comment:
Would it be possible to add this snippet at the first line, inside the **onUpdate** callback?

```
guard let `self` = self else {
   return
}
```

Benefits are:

1. You don't run all of the callback's code (if) self is released
2. By means of the `` wrapping trick, there is no need to use a keyword such as `strongSelf = self`
3. And of course, you wouldn't need to use `?` in any of the callback's lines
  
This makes sense when the callback is more than one line long. Personally, whenever the callback has a single line, and it involves a `weak self` reference, i go with the style you've used in here.